### PR TITLE
fixed typos in server error codes and messages

### PR DESCRIPTION
**Pull request by singrob**
Fixed typos in Server-Error codes and messages. 

Signed-off-by: Robert Sing <Robert.Sing@bosch-si.com>

### CODE REVIEW

File: hawkbit-core/src/main/java/org/eclipse/hawkbit/exception/SpServerError.java [hunk 1]
```diff
@@ -17,12 +17,12 @@
...
     *
     */
-    SP_REPO_GENERIC_ERROR("hawkbit.server.error.repo.genericError", "unknown error occured"),
+    SP_REPO_GENERIC_ERROR("hawkbit.server.error.repo.genericError", "unknown error occurred"),
 
     /**
     *
     */
-    SP_REPO_ENTITY_ALRREADY_EXISTS("hawkbit.server.error.repo.entitiyAlreayExists",
+    SP_REPO_ENTITY_ALREADY_EXISTS("hawkbit.server.error.repo.entityAlreadyExists",
```
Comment (schabdo) [inline hawkbit-core/src/main/java/org/eclipse/hawkbit/exception/SpServerError.java]: I would really love to fix this ugly typos in the message keys, but I'm afraid this is part of the API exposed to all clients out there in the wild. I can't tell who is relying on this specific error code shipped within a REST response. So better leave the key as it is. However fixing message text is highly appreciated!

Comment (schabdo) [reply hawkbit-core/src/main/java/org/eclipse/hawkbit/exception/SpServerError.java]: Just to be on same page: `hawkbit.server.error.repo.entityAlreadyExists` is the message key. The rest is totally fine to fix.

Comment (singrob) [reply hawkbit-core/src/main/java/org/eclipse/hawkbit/exception/SpServerError.java]: Got it, will update the PR

Comment (schabdo) [reply hawkbit-core/src/main/java/org/eclipse/hawkbit/exception/SpServerError.java]: Awesome, thanks a lot!

**Pull request merged**